alpa/util.py

```diff
@@ -863,6 +863,19 @@ def is_continuous_subset(tensor_slice, tensor_shape, row_major=True):
         return slice_shape[dim + 1:] == tensor_shape[dim + 1:]
 
 
+def infer_slice_size(slice_shape):
```

Comment:
No need for this function. `np.prod(slice_shape)` is  shorter than `infer_slice_size(slice_shape)`

Reply:
OK